lib/msal-node/src/cache/Storage.ts

```diff
@@ -66,7 +79,11 @@ export class Storage implements ICacheStorage {
      * TODO: implement after the lookup implementation
      */
     removeItem(key: string): void {
-        if (!key) return;
+        if (!key) {
+            this.emitChange();
+            return;
+        }
+        this.emitChange();
```

Comment:
The `if` seems useless, as both code paths are doing the same thing?

Reply:
Edited.